src/layout/Likert/index.tsx

```diff
@@ -1,41 +1,109 @@
   render(props: PropsFromGenericComponent<'Likert'>): JSX.Element | null {
     return <LikertComponent {...props} />;
   }
 
-  directRender(props: PropsFromGenericComponent<'Likert'>): boolean {
-    return props.node.item.layout === LayoutStyle.Table || props.overrideItemProps?.layout === LayoutStyle.Table;
+  renderSummary({
+    onChangeClick,
+    changeText,
+    summaryNode,
+    targetNode,
+    overrides,
+  }: SummaryRendererProps<'Likert'>): JSX.Element | null {
+    return (
+      <LikertSummary
+        onChangeClick={onChangeClick}
+        changeText={changeText}
+        summaryNode={summaryNode}
+        targetNode={targetNode}
+        overrides={overrides}
+      />
+    );
   }
 
-  getDisplayData(node: LayoutNode<'Likert'>, { langTools, options }: DisplayDataProps): string {
-    if (!node.item.dataModelBindings?.simpleBinding) {
-      return '';
-    }
+  renderSummaryBoilerplate(): boolean {
+    return false;
+  }
 
-    const value = String(node.getFormData().simpleBinding ?? '');
-    const optionList = options[node.item.id] || [];
-    return getSelectedValueToText(value, langTools, optionList) || '';
+  getDisplayData(): string {
+    return '';
   }
 
-  renderSummary({ targetNode }: SummaryRendererProps<'Likert'>): JSX.Element | null {
-    const displayData = this.useDisplayData(targetNode);
-    return <SummaryItemSimple formDataAsString={displayData} />;
+  hierarchyGenerator(): ComponentHierarchyGenerator<'Likert'> {
+    return this._hierarchyGenerator;
+  }
+
+  runValidations(
+    node: LayoutNode,
+    ctx: ValidationDataSources,
+    schemaErrors: ISchemaValidationError[],
+  ): FormValidations {
+    return runAllValidations(node, ctx, schemaErrors);
+  }
+
+  // This component does not have empty field validation, so has to override its inherited method
+  runEmptyFieldValidation(): ComponentValidation[] {
+    return [];
+  }
+
+  isDataModelBindingsRequired(): boolean {
+    return true;
   }
 
   validateDataModelBindings(ctx: LayoutValidationCtx<'Likert'>): string[] {
-    return this.validateDataModelBindingsSimple(ctx);
+    const [errors, result] = this.validateDataModelBindingsAny(ctx, 'Likert', ['array']);
+    if (errors) {
+      return errors;
+    }
+
+    if (result) {
+      const innerType = Array.isArray(result.items) ? result.items[0] : result.items;
+      if (!innerType || typeof innerType !== 'object' || !innerType.type || innerType.type !== 'object') {
+        return [`Likert-datamodellbindingen peker mot en ukjent type i datamodellen`];
+      }
+    }
+
+    return [];
+  }
+
+  /**
+   * Override layout validation to select a specific pointer depending on the type of group.
+   */
+  validateLayoutConfing(
+    component: CompExternalExact<'Likert'>,
+    validatate: (pointer: string, data: unknown) => ErrorObject[] | undefined,
+  ): ErrorObject[] | undefined {
+    const schemaPointer = '#/definitions/Likert';
+    return validatate(schemaPointer, component);
   }
 }
```

Comment:
Since this component is now uniquely identified by its `type`, overriding this function should no longer be necessary to give good feedback from the layout schema. So this can be removed 😃 

This is also the case for `RepeatingGroup`.

If I understand correcty, the `Group` component is now simplified with respect to panel display? If there are no additional required properties depending on if the group is displayed as a panel or not, then it can be removed for the `Group` component as well.

Either way just remove these functions, and we can check if the dev-tools warnings are precise later when we fix dev-tools again 👍